meta-mender-core/classes/mender-sdimg.bbclass

```diff
@@ -175,12 +152,27 @@ IMAGE_CMD_sdimg() {
     rm -f "$wks"
     if [ -n "${IMAGE_BOOTLOADER_FILE}" ]; then
         bootloader_align_kb=$(expr $(expr ${IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET} \* 512) / 1024)
+        bootloader_size=$(stat -c '%s' "${DEPLOY_DIR_IMAGE}/${IMAGE_BOOTLOADER_FILE}")
+        bootloader_end=$(expr $bootloader_align_kb \* 1024 + $bootloader_size)
+        if [ $bootloader_end -gt ${MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET} ]; then
+            bberror "Size of boot loader specified in IMAGE_BOOTLOADER_FILE" \
```

Comment:
Whereas this is not standardized and this is not error, I would still prefer `bootloader` over `boot loader`.